index.html

```diff
@@ -9,88 +9,103 @@
 		<script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
 	<![endif]-->
 </head>
-
 <body>
+	<div class="container">
+<!-- Menu Content  -->
+		<header>
+		S&J
+			<nav>
+			Home
+			Services
+			Contact
+			</nav>
+		</header>
 
-	<h1>You got this!  Good luck.</h1>
-
-<!-- Menu Content 
-S&J
-
-Home
-Services
-Contact
--->
-
-
-<!-- Home Page Content:
-	
-Integrity,
-Excellence,
-Progress.
+<!-- Home Page Content -->
+		<section class="cta">
+			<div class="cta-text">
+			<h1>Integrity,
+				<br>Excellence,
+				<br>Progress.</h1>
```

Comment:
remember that using br tags in a single heading is slightly bad practice. it is acceptable in small projects for lambda but know that in production you will want to just style the text in css to make it wrap over multiple lines instead of using html markup. whenever possible the html should be for semantic meaning and just hooks for the presentational studd. the css deals with the presentation and style. remember to separate your concerns and be mindful of design patterns that can make your job easier as less code = faster workflow but good code = much easier workflow

Reply:
I used <br> on a previous project and no one said a word, so thanks for the tip!